GameServer/spells/Animist/SummonAnimistFnF.cs

```diff
@@ -80,10 +80,11 @@ public override void ApplyEffectOnTarget(GameLiving target, double effectiveness
 			(m_pet.Brain as TurretBrain).IsMainPet = false;
 
 			(m_pet.Brain as IOldAggressiveBrain).AddToAggroList(target, 1);
-			(m_pet.Brain as TurretBrain).Think();
+			// (m_pet.Brain as TurretBrain).Think(); clait: Moving after the PetCount increase to fix Turrent count TODO Review
 			//[Ganrod] Nidel: Set only one spell.
 			(m_pet as TurretPet).TurretSpell = m_pet.Spells[0] as Spell;
 			Caster.PetCount++;
+			(m_pet.Brain as TurretBrain).Think(); // clait: remove this line to revert

```

Comment:
I believe this can even be removed.
A new brain whenever created will immediately `Think()`.